# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/generic/nsIFrame.cpp
# Commit: 62c1b893b282
# Full Hash: 62c1b893b282ecbc628d63fbe6844d68aa621e28
# Author: Alexander Surkov <surkov.alexander@gmail.com>
# Date: 2023-02-09 16:07:01
# Regressor Bug: 1791759
# File Overlap Count: 1
# Description:
#   Bug 1791759 - remove unused ContentRelevancyReason::DescendantOfTopLayerElement r=emilio
#   
#   A small cleanup, follow up from D159693.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D169212
# ==============================================================================

diff -r c4d83a6cced5 -r 62c1b893b282 layout/generic/nsIFrame.cpp
--- a/layout/generic/nsIFrame.cpp	Thu Feb 09 01:12:12 2023 +0000
+++ b/layout/generic/nsIFrame.cpp	Thu Feb 09 01:18:04 2023 +0000
@@ -7039,13 +7039,6 @@
                       HasSelectionInSubtree());
   }
 
-  if (!oldRelevancy ||
-      aRelevancyToUpdate.contains(
-          ContentRelevancyReason::DescendantOfTopLayerElement)) {
-    setRelevancyValue(ContentRelevancyReason::DescendantOfTopLayerElement,
-                      IsDescendantOfTopLayerElement());
-  }
-
   bool overallRelevancyChanged =
       !oldRelevancy || oldRelevancy->isEmpty() != newRelevancy.isEmpty();
   if (!oldRelevancy || *oldRelevancy != newRelevancy) {
